internal/pkg/cli/svc_deploy.go

```diff
@@ -350,14 +350,14 @@ After fixing the deployment, you can:
 func (o *deploySvcOpts) RecommendActions() error {
 	if lbMft, ok := o.appliedDynamicMft.Manifest().(*manifest.LoadBalancedWebService); ok {
 		if !lbMft.NLBConfig.IsEmpty() {
-			log.Warning("Starting with v1.33.0, Copilot will start applying a security group to your network load balancer. ",
+			log.Warning("With v1.33.0, Copilot will apply a security group to your network load balancer. ",
 				"This allows more fine-grained intra-VPC access control: ",
 				"your service won't need to allow-list the CIDR blocks of the public subnets where the NLB is deployed; ",
 				"it only needs to allow-list the NLB, specifically.\n",
 				"\n",
 				"NLB security group onboarding implies resource recreation, ",
 				"because a security group can't be added to an existing NLB that does not already have one. ",
-				"Therefore, after v1.33.0, you might see some resource recreation related to your NLB. ",
+				"Therefore, you may see some resource recreation related to your NLB. ",
```

Comment:
just making sure - we are still (successfully) outputting the nlb url at the end of `svc deploy` right?

Reply:
Yep!